prow/config.yaml

```diff
@@ -10341,6 +10341,19 @@ periodics:
       - --bare
       image: gcr.io/k8s-testimages/kubekins-e2e:v20180511-51cf02e75-master
 
+- name: ci-kubernetes-e2e-gce-device-pod-preemption-beta
```

Comment:
whoops, it does not match the config.json entry `ci-kubernetes-e2e-gce-pod-preemption-beta`

Reply:
Yup, I just noticed it from error log. I updated it now :).